pipelines/datasets/br_rf_cafir/utils.py

```
@@ -0,0 +1,145 @@
+    Args:
+        url (str): A URL do arquivo CSV a ser baixado.
+        file_name (str): O nome do arquivo a ser salvo.
+        input_folder (Path): O diretório onde o arquivo será salvo.
+        headers (dict): Cabeçalhos HTTP a serem enviados com a requisição.
+
+    Returns:
+        None
+    """
+    log(f"Downloading--------- {url}")
+    file_path = input_folder / file_name
+    response = requests.get(url)
```

Ruff (0.16.1)

[error] Probable use of `requests` call without timeout

(S113)